docs/release-notes/8.7.asciidoc

```diff
@@ -69,6 +69,8 @@ NOTE: This problem can occur when upgrading {agents} from 8.6.2 or 8.7.0 to any
 
 * Enabling malware protection on Linux hosts might cause {elastic-defend} to enter a failed state when mount points are unmounted, which will stop the integration from working. To fix this, turn off <<malware-protection,malware protection>> on the {elastic-defend} integration policy.
 
+* The **Add exceptions flyout** loads indefinitely and an out of memory error displays in the **Add exceptions flyout** if a rule's indices has a large number of fields.
```

Comment:
@e40pud @yctercero would you mind reviewing this for accuracy? TIA!

Reply:
As for the API, it depends what is the intention: to read exceptions users will use [Get](https://www.elastic.co/guide/en/security/master/exceptions-api-get-item.html) API, to create/update users will use [Create](https://www.elastic.co/guide/en/security/master/exceptions-api-create-exception-item.html)/[Update](https://www.elastic.co/guide/en/security/master/exceptions-api-update-item.html) APIs.

Maybe we can point them to the [main page](https://www.elastic.co/guide/en/security/master/exceptions-api-overview.html) for working with exceptions.

Reply:
Thanks, @e40pud! Making those changes below:
 
```suggestion
* The **Add exceptions flyout** loads indefinitely and an out of memory error displays when a rule has a large number of unmapped fields in multiple indices. To avoid this issue, use the <<exceptions-api-overview,exception APIs>> to manage exceptions.
```